### DISCUSSION TITLE
How can I make a image link that keep .png or .jpg behind?

### DISCUSSION THREAD

**Discussion by daFoggo** [zombieFox/nightTab#407] [General]
So I was trying to add image to night tab and I follow the tutorial, but when I uploaded the image on imgur, the link doesn't have .jpg behind and Night tab couldnt detect it. How I can keep it? Please help me
![image](https://user-images.githubusercontent.com/108665939/219905906-a5b5b10e-6978-4757-8acb-6f5525c3b924.png)

**Comment by zombieFox** [comment]
Hi, this is outside the control of nightTab. Seems it would be a question for the Imgur team.
